global.json

```diff
@@ -1,12 +1,4 @@
 {
-  "sdk": {
-    "version": "7.0.102",
-    "allowPrerelease": true,
-    "rollForward": "latestPatch"
-  },
-  "tools": {
-    "dotnet": "7.0.102"
-  },
```

Comment:
These shouldn't be removed.

Reply:
Oups, fixed (even if fixing the specific version is incredibly annoying, has to change it on each checkout :/)